setup.py

```diff
@@ -54,7 +54,7 @@
             "shapely>=1.6.0",
         ],
     },
-    package_data={"": ["*.csv", "*.json"]},
+    package_data={"": ["*.csv", "*.json", "*.css", "*.html"]},
```

Comment:
@oualib this should fix issue 509 right?

Reply:
yes